olympics/anaconda-project.yml

```diff
@@ -0,0 +1,42 @@
+# To reproduce: install 'anaconda-project', then 'anaconda-project run'
+name: olympics
+description: This is dummy example using a dataset containing information related to olympic athletes.
+examples_config:
+  created: 2024-03-07
+  maintainers:
+  - "jtao1"
+  labels:
+  - "hvplot"
+  deployments:
+  - command: notebook
+
+user_fields: [examples_config]
+
+channels: [defaults]
+
+packages: &pkgs
+- notebook >=6.5.2
+- python =3.11
+- hvplot >=0.8.2
+- pandas >=1.4.2an
```

Comment:
This version `1.4.2an` looks suspicious. Why is it suffixed with `an`?

Reply:
Sorry that was a typo.